Network/internet-status.2s.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+# <xbar.title>Internet Status</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Afraz Ahmadzadeh</xbar.author>
+# <xbar.author.github>afrazkhan</xbar.author.github>
+# <xbar.desc>Checks network and DNS functionality</xbar.desc>
+
+ping_timeout=1
+ping_address=9.9.9.9
+dns_address=www.quad9.net
```

Comment:
Not required to get this merged in, but something to consider. You could make these [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html) and allow users to set things based on their preferences.

Reply:
@afrazkhan what do you think of that? Could be worth it.

Reply:
Sorry, I missed this, but thanks for taking care of it for me :)